scvi/dataset/dataset.py

```diff
@@ -255,6 +265,24 @@ def _filter_genes(gene_dataset, gene_names_ref, on='gene_names'):
         subset_genes = np.array([gene_names.index(gene_name) for gene_name in gene_names_ref], dtype=np.int64)
         return gene_dataset.X[:, subset_genes], subset_genes
 
+    @staticmethod
+    def reorganize(x, genes, ordered_genes):
```

Comment:
It looks like this is only used in `CortexDataset`. Please move it there to keep it near where it's used. 

Also, please give a name that's a bit more descriptive than `reorganize`. Maybe `reorder_genes` or something?

Reply:
I'll move that and rename it. I really need to start naming my methods in a more informative way, I always come up with lousy names 😞